hw/ip/pwrmgr/data/pwrmgr_testplan.hjson

```diff
@@ -11,22 +11,225 @@
     {
       name: smoke
       desc: '''
-            Smoke test accessing a major datapath within the pwrmgr.
+            Smoke test exercising the pwrmgr state transitions.
+
+            - Brings pwrmgr out of POR.
+            - Enables wakeup.
+            - Triggers SW initiated low power transition.
```

Comment:
do we want to randomly trigger the two different kinds of low power entry? or do we always want to trigger deep sleep here?

Reply:
I will have smoke test just use the default control register. There is plenty of testing of low power settings in the wakeup test.